mcli/mcli-convert-composer-to-hf.yaml

```diff
@@ -13,7 +13,7 @@ command: |
     --hf_output_path s3://bucket/folder/hf/ \
     --output_precision bf16 \
 
-image: mosaicml/llm-foundry:2.3.1_cu121-latest
+image: mosaicml/llm-foundry:2.4.0_cu124-latest
```

Comment:
```suggestion
image: mosaicml/llm-foundry:2.4.1_cu124-latest
```

Reply:
we don't support torch 2.4.1 in foundry yet, can go in a separate PR